keycloak-event-emitter-tests/src/test/resources/META-INF/keycloak-server.json

```diff
@@ -33,7 +33,11 @@
       "eventTopic": "keycloak-event",
       "adminEventTopic": "keycloak-admin-event",
       "keycloakId": 1,
-      "datacenterId": 1
```

Comment:
You can discard changes in keycloak-server.json as this file is not used anymore with KC17 tests